### ISSUE TITLE
Defining TypeInformation with a String

### ISSUE THREAD

**Issue by twalthr** [stratosphere/stratosphere#694]
While implementing test cases for the TypeExtractor, I found it very inconvenient to define TypeInformation manually.

What do you think about a method like:

``` java
TypeInformation.parse("Tuple2<String,Tuple3<Boolean[],Integer,org.my.CustomObject>>");
```

I would like to implement it as it would help me at my current task (#683).

**Comment by StephanEwen** [comment]
+1

**Comment by fhueske** [comment]
Solved in #698
